CODEOWNERS

```diff
@@ -50,6 +50,7 @@
 /subsys/bootloader/                       @hakonfam @ioannisg
 /subsys/enhanced_shockburst/              @Raane @lemrey
 /subsys/event_manager/                    @pdunaj
+/subsys/fw_metadata/                      @hakonfam
```

Comment:
Please split this change in a separate commit.

Reply:
Can I ask why? The directory was created in that commit.

Reply:
Adding a code-owner is done in a separate commit, for example: df9ccc74c37d692127e59c246a84cea12b5c4c20
I suppose that `CODEOWNERS` is a special file and it should be clear in the log when it is changed.

Reply:
I'd rather not, since builds take long, and I want to merge this, I'll change it if I have to change something else, ok?

BTW, I don't think it's that strict: https://github.com/zephyrproject-rtos/zephyr/commit/6b21e1b7a739e95bd7a19800f72ef587e72c1a87#diff-5daf978e60e03b3975b485d5f1b449dc

Reply:
Done, but let's not run CI again